<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDetailsWidget</class>
 <widget class="QTabWidget" name="TableDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::West</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabStructure">
   <attribute name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/schema.png</normaloff>:/16/schema.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Structure</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableStructure"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabData">
   <attribute name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/table.png</normaloff>:/16/table.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Data</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnRefreshData">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/16/view-refresh.png</normaloff>:/16/view-refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableData">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabQuery">
   <attribute name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/db.png</normaloff>:/16/db.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Query</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QsciScintilla" name="txtQuery">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelQueryError">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnExecuteQuery">
          <property name="text">
           <string>&amp;Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableQueryResult"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
